databricks/koalas/frame.py

```diff
@@ -865,10 +870,19 @@ def applymap(self, func):
                    0          1
         0   1.000000   4.494400
         1  11.262736  20.857489
+
+        You can omit the type hint and let Koalas infer its type.
+
+        >>> df.applymap(lambda x: x ** 2)
+                   0          1
+        0   1.000000   4.494400
+        1  11.262736  20.857489
         """
 
         applied = []
         for idx in self._internal.column_index:
+            # TODO: We can implement shortcut theoretically since it creates new DataFrame
```

Comment:
This also will collects for every column .. we should implement separate inference logic here. will do separately.